padreader.go

```diff
@@ -13,12 +13,12 @@ func PaddedSize(size uint64) uint64 {
 	logv := 64 - bits.LeadingZeros64(size)
 
 	sectSize := uint64(1 << logv)
-	bound := ffi.GetMaxUserBytesPerStagedSector(sectSize)
+	bound := uint64(abi.PaddedPieceSize(sectSize).Unpadded())
 	if size <= bound {
 		return bound
 	}
 
-	return ffi.GetMaxUserBytesPerStagedSector(1 << (logv + 1))
```

Comment:
Should just return `abi.UnpaddedPieceSize`

Reply:
Ah, yes. Thanks.